### DISCUSSION TITLE
Which settings always load TC Collections and Environment ?

### DISCUSSION THREAD

**Discussion by Mkallday** [thunderclient/thunder-client-support#1591] [Q&A]
**I want to load all the TC collections and environments that are located in the same folder or workspace.**
 ![image](https://github.com/user-attachments/assets/6d6f3968-089e-4651-b060-156e3993ddcc)

Previously, I saved it in "Custom Location" hoping to always load all TC Collections.  TC would randomly crash and I couldn't figure how to load the collections I was working on.   **I stopped saving in the Custom Location for both User and Workspace.** 

![image](https://github.com/user-attachments/assets/35625c85-c2a0-4a8f-aed1-a8f110d52ce2)

Now, I only "Save to Workspace," but the TC Collections and Environment are always empty.

**Comment by rangav** [comment]
@Mkallday 

Create a folder name thunder-tests in your workspace and then copy collections and environments folders into it. 

Then reload vscode you should see the data

**Reply by Mkallday** [reply]
I started from scratch:
1. I created a new workspace,
2. imported a postman collection
3.  imported environment. 
____
1. Got my oAuth2 Access Token
2. Sent API request
___ 
The "crash" I mentioned previously happened again. 
![image](https://github.com/user-attachments/assets/e01e323a-ff08-4501-b49d-d78c2ba40309)

![image](https://github.com/user-attachments/assets/847c40e3-ff38-4133-9fba-bf59a57495ce)

**Comment by rangav** [comment]
Try uninstall extension and install again

**Reply by Mkallday** [reply]
Of course I did that.. I've been fiddling with settings for months

**Reply by rangav** [reply]
We will schedule a call to discuss the issue.

can you please contact us via below link
https://www.thunderclient.com/contact

**Reply by Mkallday** [reply]
![image](https://github.com/user-attachments/assets/b8ea6ef5-3f65-44ef-9775-fe26cb8952b7)
